![alt text](image.png)

![alt text](image-1.png)

- Tentando contornar o erro acima
![alt text](image-2.png)
![alt text](image-3.png)

- Criar arquivo docker
![alt text](image-4.png)
![alt text](image-5.png)

- docker compose up --build
![alt text](image-6.png)

- o docker compose parou nessa última linha e já se vão uns 10 minutos
![alt text](image-7.png)

- localhost:9001
![alt text](image-8.png)
![alt text](image-9.png)

- dbeaver
![alt text](image-10.png)

- resolvendo falha na conexão
![alt text](image-11.png)
![alt text](image-12.png)

- erro (triste) no minio
![alt text](image-13.png)
- não estou conseguindo resolver o erro no minio
![alt text](image-14.png)

- test-minio.py rodou!!!
![alt text](image-15.png)
![alt text](image-16.png)
![alt text](image-17.png)

-sql 
![alt text](image-18.png)


- criação .env
![alt text](image-19.png)

-
-
-

- client clickhouse
![alt text](image-20.png)

- data processing
![alt text](image-21.png)

- minio_client
![alt text](image-22.png)

- servidor app.py
![alt text](image-23.png)

